adapters/qt/qt_test.go

```diff
@@ -0,0 +1,20 @@
+package qt
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/v2/adapters/adapterstest"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderQT, config.Adapter{
+		Endpoint: "https://endpoint1.qt.io/pserver"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
```

Comment:
Nitpick: It's not necessary to use a real url for tests. Might be easier to use an obviously fake value so there's no pressure to keep it updated.